.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: java
 sudo: false
-script: mvn clean test
+script: mvn clean test -DrunSuite=**/GoogleOAuthPlugInTestSuite.class
```

Comment:
Out of curiosity, what advantage do you see of specifying a suite vs. allowing all unit tests to be run?

Reply:
Would prefer to configure the plugin suites which get executable via a pom file configuration, see: https://stackoverflow.com/questions/11762801/run-junit-suite-using-maven-command

Reply:
If some of the tests are in Beta phase, one shouldn't add them to suite and thus, No failure will happen. 
Apart from that you can run all tests in single suite at a time.
E.g: CredentialsOAuthTestSuite will run all tests inside: com.google.jenkins.plugins.credentials.oauth
and UtilTestSuite will run all tests inside com.google.jenkins.plugins.util package.
Moreover GoogleOAuthPlugInTestSuite is collection of all test suites,that is equivalent to running mvn test.

Reply:
That makes sense. Thanks!